python/cudf/bindings/sort.pyx

```diff
@@ -38,7 +38,12 @@ cpdef apply_order_by(in_cols, out_indices, ascending=True, na_position=1):
     check_gdf_compatibility(out_indices)
     cdef gdf_column* output_indices = column_view_from_column(out_indices)
 
-    cdef int flag_nulls_are_smallest = na_position
+    if na_position == 1:
+        null_sort_behavior_api = 'null_as_smallest'
+    else :
+        null_sort_behavior_api = 'null_as_largest'
+
+    cdef gdf_context* context = create_context_view(0, 'sort', 0, 0, 0, null_sort_behavior_api)
```

Comment:
You need to free this as it mallocs `sizeof(gdf_context)`

Reply:
done